src/contracts/interfaces/IRegistryCoordinator.sol

```diff
@@ -23,15 +31,21 @@ interface IRegistryCoordinator {
         // start taskNumber from which the  operator has been registered
         uint32 fromTaskNumber;
         // indicates whether the operator is actively registered for serving the middleware or not
-        Status status;
+        OperatorStatus status;
```

Comment:
is the `fromTaskNumber` useful/needed/sufficient?
we should revisit why we have this / if it is still useful / what data we actually want or need here.

Reply:
I think you mentioned offline that it might be used in payments. IMO, let's not keep track of this because our payments mechanism ideas have changed. By the time we get to sophisticated fraud proofs, we won't need this. wdyt?

Reply:
I think you have a better handle on where we're at with EigenDA payments. I am OK leaving for now, it seems fine to remove later, maybe add a TODO to the code about revisiting this for now?

Reply:
Agreed

Reply:
https://github.com/Layr-Labs/eigenlayer-contracts/pull/76/commits/f97ac404f08237474a68639a1037f92243dffc0f